### PR TITLE
Notify bugfixes

### PR DESCRIPTION
**Pull request by dszlachta**
- Fixes issues with Notify
- Delegates CID generation for chunks (when notifying about new chunks) to IPFS. I couldn't get the same CID otherwise.

### CODE REVIEW

File: src/apps/chifra/internal/scrape/scrape_prepare.go [hunk 1]
```diff
@@ -22,6 +23,16 @@ import (
 func (opts *ScrapeOptions) Prepare() (ok bool, err error) {
 	chain := opts.Globals.Chain
 
+	// Notify feature requires IPFS daemon to be running. We send
+	// a simple query (Version) to check if it's there
+	if ok, _ := NotifyConfigured(); ok {
+		sh := shell.NewShell(config.GetPinning().LocalPinUrl)
+		_, _, err = sh.Version()
```
Comment (tjayrush) [inline src/apps/chifra/internal/scrape/scrape_prepare.go]: Can we use this: config.IpfsRunning()?

Comment (dszlachta) [reply src/apps/chifra/internal/scrape/scrape_prepare.go]: Thanks!

**Comment by tjayrush** [comment]
@dszlachta Is this ready to merge?

**Comment by dszlachta** [comment]
Yep 👍

**Pull request merged**